SPECS-EXTENDED/perl-Razor-Agent/perl-Razor-Agent.spec

```diff
@@ -0,0 +1,313 @@
+Summary:        Use a Razor catalogue server to filter spam messages
+Name:           perl-Razor-Agent
+Version:        2.85
+Release:        37%{?dist}
+License:        Artistic 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://razor.sourceforge.net/
+Source:         https://downloads.sourceforge.net/project/razor/razor-agents/%{version}/razor-agents-%{version}.tar.bz2#/%{name}-%{version}.tar.bz2
+Patch0:         razor-agents-2.85-use-sha-not-sha1.patch
+Patch1:         perl-Razor-Agent-2.85-mandir.patch
+Patch2:         perl-Razor-Agent-2.85-parallel-make.patch
+BuildRequires:  gcc
+BuildRequires:  perl-devel
+BuildRequires:  perl-generators
+BuildRequires:  perl(Digest::SHA1)
+BuildRequires:  perl(ExtUtils::MakeMaker)
+BuildRequires:  perl(Net::DNS)
+BuildRequires:  perl(Time::HiRes)
+BuildRequires:  perl(URI)
+Requires:       perl(:MODULE_COMPAT_%(eval "`perl -V:version`"; echo $version))
+Requires:       perl(Net::DNS)
+# Filter the Perl extension module
+%{?perl_default_filter}
+
+%description
+Vipul's Razor is a distributed, collaborative, spam detection and
+filtering network. Razor establishes a distributed and constantly
+updating catalogue of spam in propagation. This catalogue is used
+by clients to filter out known spam. On receiving a spam, a Razor
+Reporting Agent (run by an end-user or a troll box) calculates and
+submits a 20-character unique identification of the spam (a SHA
+Digest) to its closest Razor Catalogue Server. The Catalogue Server
+echos this signature to other trusted servers after storing it in its
+database. Prior to manual processing or transport-level reception,
+Razor Filtering Agents (end-users and MTAs) check their incoming mail
+against a Catalogue Server and filter out or deny transport in case of
+a signature match. Catalogued spam, once identified and reported by
+a Reporting Agent, can be blocked out by the rest of the Filtering
+Agents on the network.
+
+%prep
+%setup -q -n razor-agents-%{version}
```

Comment:
Use autosetup

Reply:
I think as per line 46, which is not specifiying the `-p` level. The patch may not apply.